docs/resources/samples/convert-csv.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Convert CSV files to Excel workbooks'
+description: 'Learn how to use Office Scripts and Power Automate to create .xlsx files from .csv files.'
+ms.date: 07/19/2021
+localization_priority: Normal
+---
+
+# Convert CSV files to Excel workbooks
+
+Many services export data as comma-separated value (CSV) files. This solution automates the process of converting those CSV files to Excel workbooks in the .xlsx file format. It uses a [Power Automate](https://flow.microsoft.com) flow to find .csv files in a OneDrive folder and an Office Script to copy the data from the .csv file into a new Excel workbook.
+
+## Solution
+
+1. Store the .csv files and a blank "Template" .xlsx file in a OneDrive folder.
+1. Create an Office Script to parse the CSV data into a range.
+1. Create a Power Automate flow to read the .csv files and pass their contents to the script.
+
+## Sample files
+
+Download <a href="https://github.com/OfficeDev/office-scripts-docs/blob/master/docs/resources/samples/convert-csv-example.zip?raw=true">convert-csv-example.zip</a> to get the template .xlsx file and two sample .csv files. Extract the files into a folder in your OneDrive. This sample assumes the folder is named "output".
```

Comment:
This link doesn't work for me – it goes to a GitHub 404 page. Am I doing something wrong?

Reply:
No, links to raw files need to include a branch. I forgot about this quirk and should have specified to use the file in the PR instead of the link.